site/app/templates/forum/ShowCategories.twig

```diff
@@ -0,0 +1,119 @@
+<div class="content forum_content">
```

Comment:
This file is an almost direct copy of `site/app/templates/forum/CategoriesForm.twig`. From our conversation I thought we were planning on moving the categories to it's own page exclusively (and get rid of the popup). We should just modify the `CategoriesForm.twig` to be it's own page rather than making this a new file. I can envision this page could become a forum setting page in the future.

Reply:
I remembered something along the lines of having the popup where it is, only adding conditions to the rendering of that twig file to use/skip the popup. But on working with the file I realized its construction is such that this was not possible, hence skipped and created a new file instead! Would you suggest to entirely remove the popup? I would only need to make a few changes for that! :)

Reply:
Yeah, let's confirm tomorrow morning (I would personally like to do away with the popup if we're going to have a page). But, if we were to keep both, we could have a twig file that would include everything from line `ShowCategories.twig:15` and down. This would then be able to be included in the popup `CategoriesForm.twig` and the corresponding twig file for the categories page. But let's discuss tomorrow.